processor/src/main/java/com/linecorp/decaton/processor/runtime/SubscriptionBuilder.java

```diff
@@ -59,8 +59,8 @@
         presetRetryProducerConfig.put(ProducerConfig.LINGER_MS_CONFIG, "100");
     }
 
-    @Setter(AccessLevel.NONE)
-    private ProcessorProperties.Builder<ProcessorProperties> propertiesBuilder;
+    private final ProcessorProperties.Builder<ProcessorProperties> propertiesBuilder =
```

Comment:
> avoid overwriting properties in SubscriptionBuilder - I can't see any good reason to do it

The intention is to provide a way to "clear" (reset) something set previously. So basically these methods taking collection of items replaces existing set rather than adding.
To keep this behavior consistent to other methods like `consumerConfig` and backward, I think we shouldn't change this. 
I'm happy to add another kind of method like `addProperties()` (`addProperty()`) which supports it and being more explicit tho.

Reply:
I don't think that's what one would expect of a builder, but happy to oblige with your design constraints